scripts/build/docker/Dockerfile.template

```diff
@@ -12,6 +12,10 @@ RUN echo "deb http://archive.ubuntu.com/ubuntu precise-backports main restricted
 RUN echo "deb http://ftp.debian.org/debian jessie-backports main" >> /etc/apt/sources.list
 <% } %>
 
+# Workaround: Install a newer version of `tar` to make fpm in electron-builder work again
+RUN echo "deb http://ftp.de.debian.org/debian wheezy main contrib non-free" >> /etc/apt/sources.list
```

Comment:
Should we just use `http://ftp.debian.org/` as in the previous lines, rather than selecting the `de`-specific subdomain?

Reply:
Done